app/views/layouts/_header.html.erb

```diff
@@ -4,33 +4,35 @@
       <span class="navbar-toggler-icon"></span>
     </button>
 
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
-      <ul class="navbar-nav mr-auto">
-        <li class="nav-item active background-blue">
-          <%= link_to new_user_registration_path, class: 'nav-link nav-icon nav-first' do %>
-            <%= image_tag "BW- CREATE USER ICON.svg", alt:"Create a User" %><p>CREATE USER</p>
-          <% end %>
-        </li>
-        <li class="nav-item active background-slate">
-          <%= link_to new_carrier_path, class: 'nav-link nav-icon nav-second' do %>
-            <%= image_tag "BW- ADD USER ICON.svg", alt:"Add an Item" %><p>ADD ITEM</p>
-          <% end %>
-        </li>
-        <li class="nav-item active background-dark-purple">
-          <a class="nav-link nav-icon nav-third" href="/"><%= image_tag "BW- CART ICON.svg", alt:"Cart" %><p>CHECK IN/OUT</p></a></li>
-        <li class="nav-item active background-purple">
-          <a class="nav-link nav-icon nav-fourth" href="/"><%= image_tag "BW - INVENTORY ICON.svg", alt:"View Inventory" %><p>CARRIERS</p></a></li>
-        <li class="nav-item dropdown">
-          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            Quick Access
-          </a>
-          <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-            <a class="dropdown-item" href="#">Action</a>
-            <a class="dropdown-item" href="#">Another action</a>
-            <div class="dropdown-divider"></div>
-            <a class="dropdown-item" href="#">Something else here</a>
-          </div>
-        </li>
-      </ul>
-    </div>
+    <% if signed_in? %>
```

Comment:
What are your thoughts, would it make sense to use two different layouts, one when a user is not authenticated and one when a user is?

Reply:
I'd be in favor of that. That is how I would do it on a greenfield project. I didn't want to do that big of a change in this branch. Probably worth creating an issue for that.

Reply:
Sure thing, do it in a separate PR!